bertopic/_bertopic.py

```diff
@@ -756,6 +751,50 @@ def _cluster_embeddings(self,
             f"Clustered UMAP embeddings with {self.clustering_method.upper()}")
         return documents, probabilities
 
+    def get_best_hdbscan_model(self, embeddings, exclude=-1):
+        mcs = self.min_topic_size
+        # todo: a better structure for search space
+        search_space = {
+            'cluster_selection_method': ['eom', 'leaf'],
+            'min_samples': set(map(int, [1, 5, 10, mcs * 0.25, mcs * 0.33, mcs * 0.5, mcs * 0.66,
+                                         mcs, mcs * 1.33, mcs * 1.5, mcs * 1.75, mcs * 2,
+                                         mcs * 2.25, mcs * 2.5, mcs * 2.75, mcs * 3]))
+        }
+        arg_names = search_space.keys()
+        arg_values = list(product(*search_space.values()))
+        logger.info(f"Searching started for best parameters with mcs={mcs} "
+                    f"totally {len(arg_values)} runs..")
+        best_score = -1
```

Comment:
Maybe not a case for calinski_harabasz_score but better to set the initial value to minus inf.
```suggestion
        best_score = float("-inf")
```